Sources/BraintreeThreeDSecure/BTThreeDSecureRequest.m

```diff
@@ -227,19 +235,21 @@ - (void)startRequest:(BTPaymentFlowRequest *)request configuration:(BTConfigurat
 }
 
 - (void)processLookupResult:(BTThreeDSecureResult *)lookupResult configuration:(BTConfiguration *)configuration {
-    if (!lookupResult.lookup.requiresUserAuthentication) {
-        [self.paymentFlowDriverDelegate onPaymentComplete:lookupResult error:nil];
+    if (!lookupResult.lookup.isThreeDSecureVersion2) {
```

Comment:
Let's match Android and iOS v6 here ([see discussion](https://github.com/braintree/braintree_ios/pull/1046/files#r1213205061))